packages/ui-components/src/README.md

```diff
@@ -1,4 +1 @@
-# Components
-
-- [Badge](Badge/README.md)
-- [Tooltip](Tooltip/README.md)
+# ui-components
```

Comment:
I guess I ultimately agree with @laurenbarker about the maintainability of the README documentation. Since we are going to move all documentation to Storybook, we might as well chop all README documentation out. I hope there is a way to generate something in the repo in terms of documentation, but until we get there this file doesn't make any sense anymore.

Reply:
That being said, we should put A LOT more links to where Storybook is if it's going to be the only source of documentation for the components.

Reply:
+1 to adding a link to https://core-components.crdb.io/ in this readme